blob/src/main/java/io/crate/http/netty/HttpBlobHandler.java

```diff
@@ -74,20 +55,22 @@
 public class HttpBlobHandler extends SimpleChannelUpstreamHandler implements
         LifeCycleAwareChannelHandler {
 
-    public static final String CACHE_CONTROL_VALUE = "max-age=315360000";
-    public static final String EXPIRES_VALUE = "Thu, 31 Dec 2037 23:59:59 GMT";
-    public static final String BLOBS_ENDPOINT = "/_blobs";
-    public static final Pattern pattern = Pattern.compile(String.format("^%s/([^_/][^/]*)/([0-9a-f]{40})$", BLOBS_ENDPOINT));
-    private static final ESLogger logger = Loggers.getLogger(HttpBlobHandler.class);
+    private static final String CACHE_CONTROL_VALUE = "max-age=315360000";
+    private static final String EXPIRES_VALUE = "Thu, 31 Dec 2037 23:59:59 GMT";
+    private static final String BLOBS_ENDPOINT = "/_blobs";
+    public static final Pattern BLOBS_PATTERN = Pattern.compile(String.format("^%s/([^_/][^/]*)/([0-9a-f]{40})$", BLOBS_ENDPOINT));
+    private static final Matcher BLOBS_MATCHER = BLOBS_PATTERN.matcher("");
```

Comment:
ok, seems ok in that case, though i don't especially like it.
please add your above comment as a note, then ack